### PR TITLE
(feat) O3-3839 Allowing of printing lab request

### PR DESCRIPTION
**Pull request by CynthiaKamau**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary

Allowing of printing lab request

## Screenshots

https://github.com/user-attachments/assets/f46ce555-baed-44af-a521-186af12e27e8

### CODE REVIEW

File: packages/esm-patient-orders-app/src/components/orders-details-table.component.tsx [hunk 1]
```diff
@@ -347,8 +347,8 @@ const OrderDetailsTable: React.FC<OrderDetailsProps> = ({ patientUuid, showAddBu
                       {showPrintButton && (
                         <Button
                           kind="ghost"
-                          renderIcon={(props) => <PrinterIcon {...props} size={16} />}
-                          iconDescription={t('print', 'Print')}
+                          renderIcon={PrinterIcon}
+                          iconDescription="Add vitals"
```
Comment (denniskigen) [inline packages/esm-patient-orders-app/src/components/orders-details-table.component.tsx:351]: This is incorrect.

File: packages/esm-patient-orders-app/src/config-schema.ts [hunk 2]
```diff
@@ -6,8 +6,15 @@ export const configSchema = {
     _description: 'The encounter type of the encounter encapsulating orders',
     _default: '39da3525-afe4-45ff-8977-c53b7b359158',
   },
+  showPrintButton: {
+    _type: Type.Boolean,
+    _default: false,
+    _description:
+      'Determines whether or not to display the Print button in the orders. If set to true, a Print button gets shown in both the orders table headers. When clicked, this button enables the user to print out the contents of the table',
```
Comment (denniskigen) [inline packages/esm-patient-orders-app/src/config-schema.ts:13]:
```suggestion
      'Determines whether or not to display a Print button in the Orders details table. If set to true, a Print button gets shown in both the orders table headers. When clicked, this button enables the user to print out the contents of the table',
```

**Comment by denniskigen** [comment]
Also, this is technically a new feature, so the appropriate conventional commit label should be `(feat)`.

**Comment by CynthiaKamau** [comment]
> Also, this is technically a new feature, so the appropriate conventional commit label should be `(feat)`.

it already existed, but was set to false by default, i just added a config to expose it

**Comment by CynthiaKamau** [comment]
> I have some concerns about this. I believe the end product was for the user to be able to print the order request details. Typically, printing the list of orders might not add much value but printing for example the lab order details with the tests as well as results offer much more value. Looking at the ticket, it doesn't say much about what needs to be achieved so maybe we should get clarification first.

i have pinged the design team, il be updating the layout after a design is created

**Review by pirupius — APPROVED** [review]

**Pull request merged**